<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerWindow</class>
 <widget class="QMainWindow" name="ViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multispectral Image Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gerbil.qrc">
    <normaloff>:/logoicon</normaloff>:/logoicon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="ImgGradLayout">
    <item>
     <layout class="QHBoxLayout" name="toolbarLayout">
      <item>
       <widget class="QPushButton" name="nonmarkButton">
        <property name="text">
         <string>Show Unlabeled</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="markButton">
        <property name="text">
         <string>Show Labeled</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ignoreButton">
        <property name="text">
         <string>Ignore Labels</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="toolTip">
         <string>Add to current label</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="gerbil.qrc">
          <normaloff>:/toolbar/add</normaloff>:/toolbar/add</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remButton">
        <property name="toolTip">
         <string>Remove from current label</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="gerbil.qrc">
          <normaloff>:/toolbar/remove</normaloff>:/toolbar/remove</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sliceButton">
        <property name="text">
         <string>View Slice</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="multi_img_viewer" name="viewIMG" native="true"/>
    </item>
    <item>
     <widget class="multi_img_viewer" name="viewGRAD" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="sliceDock">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Image Slice</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="sliceLayout">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Update</string>
         </property>
         <property name="icon">
          <iconset resource="gerbil.qrc">
           <normaloff>:/toolbar/left</normaloff>:/toolbar/left</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="markerSelector">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
         <property name="minimumContentsLength">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="icon">
          <iconset resource="gerbil.qrc">
           <normaloff>:/toolbar/clear2</normaloff>:/toolbar/clear2</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="SliceView" name="sliceLabel" native="true">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>multi_img_viewer</class>
   <extends>QWidget</extends>
   <header>multi_img_viewer.h</header>
  </customwidget>
  <customwidget>
   <class>SliceView</class>
   <extends>QWidget</extends>
   <header>viewport.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gerbil.qrc"/>
 </resources>
 <connections/>
</ui>
